src/Symfony/Component/Templating/DelegatingEngine.php

```diff
@@ -11,6 +11,8 @@
 
 namespace Symfony\Component\Templating;
 
+trigger_deprecation('symfony/templating', '6.4', '"%s" is deprecated since version 6.4 and will be removed in 7.0. Use Twig instead.', DelegatingEngine::class);
```

Comment:
Please also flag all classes as `@deprecated` so IDEs and static analyzers may pick up the deprecation.